apps/foundation/pages/platforms/academy/software.tsx

```diff
@@ -0,0 +1,69 @@
+import { FC } from 'react'
+
+import { Box, Heading, Stack, Text } from '@chakra-ui/react'
+import { GetStaticPropsContext, InferGetStaticPropsType } from 'next'
+import { useRouter } from 'next/router'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+
+import { ASSETS_URL } from '@wsvvrijheid/config'
+import { StrapiLocale } from '@wsvvrijheid/types'
+import { Container, Hero } from '@wsvvrijheid/ui'
+
+import { Layout } from '../../../components'
+import i18nConfig from '../../../next-i18next.config'
+
+type SoftwareProps = InferGetStaticPropsType<typeof getStaticProps>
+
+const Platforms: FC<SoftwareProps> = ({ title, content }) => {
+  const { locale } = useRouter()
+
+  const contentBody = content[locale as StrapiLocale]
+
+  return (
+    <Layout seo={{ title }}>
```

Comment:
If use use Hero component on the page, Layout should take `isDark` prop that changes header menu colors.

```suggestion
    <Layout seo={{ title }} isDark>
```